pycode/memilio-epidata/memilio/epidata_test/test_epidata_getDataIntoPandasDataFrame.py

```diff
@@ -280,14 +280,7 @@ def test_cli_correct_raise_exit(self, mock_stderr):
                 mock_stderr.getvalue(),
                 r"unrecognized arguments")
 
-        test_args = ["prog", '--make-plot']
-        with patch.object(sys, 'argv', test_args):
-            with self.assertRaises(SystemExit) as cm:
-                gd.cli("divi")
-            self.assertRegexpMatches(
-                mock_stderr.getvalue(),
-                r"unrecognized arguments")
-
+        # test wrong date format
```

Comment:
It does not test wrong date format, does it? It rather tries to get data for dates which are not available, right?

Reply:
Correct format woud be `'2020-03-24'`. All other formats raise errors.
If the start date is not available, it is simply changed to the first available date (202-04-24) but this should not raise any errors.